lua/orgmode/objects/date.lua

```diff
@@ -53,10 +59,13 @@ local Date = {
   end,
 }
 
+---@alias DateTbl { year: number, month: number, day: number, hour: number, min: number }
```

Comment:
Let's prefix all of the types with `Org` just to avoid potential clash with other plugins.

Reply:
Actually in the end I didn't used these aliases anymore. Removed.